src/Sylius/Bundle/CoreBundle/Tests/Controller/SitemapControllerTest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\CoreBundle\Tests\Controller;
+
+use Lakion\ApiTestCase\XmlApiTestCase;
+
+/**
+ * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
+ */
+class SitemapControllerTest extends XmlApiTestCase
+{
+    public function testShowActionResponse()
+    {
+        $this->purgeDataBase();
+        $this->loadFixturesFromFile('Product.yml');
```

Comment:
File name should not be camelCased. Also, this fie should be put to some folder. Resources for example. Or catalogue .